librz/core/cmd_system.c

```diff
@@ -0,0 +1,123 @@
+static char *system_cmd_new(int argc, const char **argv) {
+	if (argc == 1) {
+		return strdup(argv[0]);
+	}
+
+	RzStrBuf *sb = rz_strbuf_new(argv[0]);
+	if (!sb) {
+		return NULL;
+	}
+
+	for (int i = 1; i < argc; ++i) {
+		rz_strbuf_append(sb, " \"");
+		const char *arg = argv[i];
+		if (strchr(arg, '"') || strchr(arg, '\\')) {
+			size_t p, s, len = strlen(arg);
+			for (p = 0, s = 0; p < len; ++p) {
+				if (arg[p] == '\\' || arg[p] == '"') {
+					rz_strbuf_append_n(sb, arg + s, p - s);
+					rz_strbuf_append(sb, "\\");
+					s = p;
+				}
+			}
+			arg += s;
+		}
+		rz_strbuf_appendf(sb, "%s\"", arg);
+	}
+
+	return rz_strbuf_drain(sb);
+}
```

Comment:
It remains to see whether this works well on windows... I remember that I wanted to switch all this `!` stuff to RzSubprocess API.

Reply:
I can add a comment to refactor this later.